visualdl/utils/image.h

```diff
@@ -65,8 +66,8 @@ static void NormalizeImage(Uint8Image* image,
     if (finite) {
       for (int j = 0; j < depth; j++) {
         float v = values(j, i);
-        image_min = std::min(image_min, v);
-        image_max = std::max(image_max, v);
+        image_min = (std::min)(image_min, v);
```

Comment:
why here need `(std::min)` ?

Reply:
In Windows platform `unsupported/Eigen/CXX11/Tensor` includes `windows.h` , which defines `max/min` as macro. Put parentheses around function name prevents the macro expansion.

Reply:
Get it. Better to leave the above answer as a comment here, because this statement might be a little strange for Linux c++ developers.

Reply:
I update the PR to define `NOMINMAX` in CMake, so that `windows.h` will not define the min/max macros.